src/components/scrollButton/index.js

```diff
@@ -42,7 +43,7 @@ export default class ScrollButton extends Component {
         }}
         aria-label="Back to top"
       >
-        <FontAwesomeIcon icon={['fas', 'arrow-up']} />
+        <FontAwesomeIcon icon={faArrowUp} />
```

Comment:
Ble vi enige om array-syntaksen nå som vi fikk CSSen til å funke?

Reply:
Kan godt importere ikonene hver for seg istedet for hele pakkene

Reply:
Om vi importerer ikonene overalt eller legger alle ikonene inn i library-et en-og-en er litt samme for meg, bare bestem dere for hva dere vil gjøre og gjør det likt over hele applikasjonen :)

Reply:
cc @Staldvik